docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -47,7 +46,7 @@ Software
 .. prompt:: bash $
 
    brew tap stepanstipl/noop
-   brew install createrepo dpkg libffi openssl
+   brew install createrepo dpkg
```

Comment:
:question: I'm inclined to remove these instructions too and say that the release process must be performed from a standard Centos7 machine....and I think we'd both like to see the Vagrant dev environment re-instated. I'll file a followup issue if you agree.